### ISSUE TITLE
P03-C04 Detailed Feedback

### ISSUE THREAD

**Issue by madanvikram** [zackchase/mxnet-the-straight-dope#137]
Overall the tutorial was easy to follow and well written. Here is my detailed feedback:

High-level comments:
1) To better align the ‘Scratch’ and ‘Gluon’ example, let’s create a dropout variable to be used in the training loop and allow a dropout % parameter to be passed to the net (versus hard coding the dropout % in net)

Areas that require clarification:
1) We should clarify (and perhaps cover in-depth in P03-C03) the following: “…but scale down their values, essentially averaging the various dropped out nets.”
2) We should expand on the following and further explain in the Gluon context: “DropOut is a special kind of layer because it behaves differently when training and predicting.”
3) I found the following sentence difficult to parse: “For example, when we generate adversarial examples (a topic we'll investigate later) we may want to record, but for the model to behave as in predict mode.”

Typos / minor changes:
1) Change ‘implementation by hand’ to ‘implementation from scratch’
2) In the model, the phrase “Adding first hidden layer” is repeated
3) Remove parentheses around (with autograd.record(train_mode=False):)
